### ISSUE TITLE
Small enhancement 720p and 1080p detection

### ISSUE THREAD

**Issue by ratoaq2** [guessit-io/guessit#319]
`guessit 2.0.5`

720pHD and 1080pHD are not detected

```
guessit -t episode "Show.Name.(s01e10).LEVEL.TEN.|.GAME.OVER.-.YouTube.Red.(720pHD).mp4-[Zeus]"
For: Show.Name.(s01e10).LEVEL.TEN.|.GAME.OVER.-.YouTube.Red.(720pHD).mp4-[Zeus]
GuessIt found: {
    "title": "Show Name", 
    "season": 1, 
    "episode": 10, 
    "episode_title": "LEVEL TEN GAME OVER - YouTube Red", 
    "release_group": "720pHD", 
    "container": "MP4", 
    "type": "episode"
}
```

If you agree, I could create a PR with a small fix for this together with some release_names to the existing tests

**Comment by Toilal** [comment]
I agree if it doesn't break others unit tests.

**Comment by ratoaq2** [comment]
This was fixed by #324
